README.md

```diff
@@ -23,6 +23,18 @@ Unleash is the most popular open source solution for feature flagging on GitHub.
 
 <br/>
 
+## Start with Unleash Enterprise
```

Comment:
@FredrikOseberg I wouldn't make this a separate section, but include it as part of **Getting Started with Unleash**. We could phrase it something like: To start with Unleash Enterprise, [request a free trial](..). This gives you access to a hosted instance where you can test features such as ...

Then: If you want to start with Unleash Open Source, ... 

Happy to make a commit with the proposed changes if you'd like

Reply:
@melindafekete Feel free to make the changes.

Reply:
@FredrikOseberg made some changes here 04cea6055ab58b30b972816ef31bfeedf42ad4eb wdyt?

Reply:
Nice! Thank you.